### PR TITLE
ZNP-25: Creating schedule appointments

### PR DESCRIPTION
**Pull request by RogerPino**
ReminderAppointments component was created and implemented

### CODE REVIEW

File: src/containers/ReminderAppointments/components/ReminderDateFilter/index.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Icon, Square, Text, Image } from "@chakra-ui/react";
+import { GrFormPrevious } from "react-icons/gr"
+import { FcNext } from "react-icons/fc"
+import arrow_previous from 'assets/icons/arrow_previous.svg';
+import arrow_next from 'assets/icons/arrow_next.svg';
+
+
+export type TReminderDateFilter = {
+}
```
Comment (acnicolasdc) [inline src/containers/ReminderAppointments/components/ReminderDateFilter/index.tsx:8]: If the T is empty, why did you create it?

File: src/containers/ReminderAppointments/components/index.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Box, Flex, Square, Heading, VStack, StackDivider, Button } from "@chakra-ui/react";
+import ReminderDateFilter from 'containers/ReminderAppointments/components/ReminderDateFilter';
+import ReminderAppointmentItem from 'containers/ReminderAppointments/components/ReminderAppointmentItem';
+
+export type TReminderAppointments = {
+}
```
Comment (acnicolasdc) [inline src/containers/ReminderAppointments/components/index.tsx:7]: Same here

File: src/containers/ReminderAppointments/components/ReminderAppointmentItem/index.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import { HStack, Icon, Text, VStack, Box } from "@chakra-ui/react";
+import { RiCheckboxBlankCircleFill } from "react-icons/ri"
+
+
+export type TReminderAppointmentItem = {
+}
+
+export default function ReminderAppointmentItem() {
+    return (
+        <>
```
[the remark this answers is not in the document]
Comment (RogerPino) [reply src/containers/ReminderAppointments/components/ReminderAppointmentItem/index.tsx:11]: Resolved
I only did it while structuring the component

File: src/containers/ReminderAppointments/components/ReminderDateFilter/index.tsx (hunk 1 above)
Comment (RogerPino) [reply src/containers/ReminderAppointments/components/ReminderDateFilter/index.tsx:8]: I left it thinking that maybe we would use it later. 
Removed

File: src/containers/ReminderAppointments/components/index.tsx (hunk 2 above)
Comment (RogerPino) [reply src/containers/ReminderAppointments/components/index.tsx:7]: Resolved

**Pull request merged**